### ISSUE TITLE
Programmplaner: "Kommt noch"-Button -> Spielziele

### ISSUE THREAD

**Issue by GWRon** [TVTower/TVTower#270]
Urgestein Dalli schlug mir im Chat vor, dass der "kommt noch"-Button ja einen Bildschirm mit "Spielzielen" hervorrufen koennte.

> Chef will Sammy, dann könnte das da drin stehen
> aber: Die zwischenstände gibts trotzdem nur bei Betty
> aber Chef will zum Beispiel: Mehr Sendegebiet -> Dafür gibts dann Image oder Bissl Asche

Denke, wir koennten da auch das allgemeine Spielziel ("Liebe" - oder bei Missionen, was anderes) mit angeben.

Fraglich ist, ob es einen eigenen Bildschirm "fuellen" kann.

**Comment by nittka** [comment]
Wie grenzt man dann "Erfolge" von Spielzielen ab?

**Comment by GWRon** [comment]
Erfolge = freiwillig erreichte Achievements

Spielziele: "muessen" erreicht werden... bzw teilweise (fuer "einen Boni oben drauf")

**Comment by nittka** [comment]
Erfolgs"bedingungen" müssen ja auch erreicht werden für den Bonus. Ich glaube, mir ist der technische Unterschied zwischen den aktuellen Erfolgen und den geplanten Spielzielen (und möglicherweise "Missionen") klar. Ich sehe nur keinen großen Mehrwert darin, wenn es einen Erfolg "5Mio Zuschauer" und ein Spielziel "5Mio Zuschauer" gibt (oder ein Spielziel 10Mio in 15 Tagen und dann eine Erfolg 5Mio der "unerwartet" zwischendurch erreicht wird).

Mir stellt sich als die Frage, ob es nicht sinnvoller wäre, am Anfang zwischen Endlosspiel mit "unbekannten" Erfolgen und "Missionsspiel" (Betty heiraten, 50 % Reichweite in 25 Tage, ...) zu wählen. In diesem Fall wäre aber die Verwendung des weiteren Buttons nicht nötig und selbst im Parallelbetrieb (Spielziel+Erfolge) würde der existierende Button Erfolge (mit angepasster Benennung) komplett ausreichen.

**Comment by GWRon** [comment]
War auch nur ein Vorschlag.


Eventuell sollte der Bereich fuer "Umfragen" oder so genommen werden (Umfragen durchfuehren, Ergebnisse ansehen). Oder - um die "Statistiken" nicht zu ueberfrachten, "Eigenproduktionen" als eigenen Bereich einfuehren - mit Auswertung usw.

**Comment by GWRon** [comment]
Dalli:
>Huhu,
> also die Abgrenzung zwischen Erfolgen und "Aufgaben vom Chef" ist der, dass diese Aufgaben, bei nicht erreichen auch negative Folgen haben müssen.
> Dies würde etwas pepp ins Spiel bringen, zum Beispiel, wenn der Chef raus poltert, dass innerhalb von 7 Tagen 2 Live-Übertragungen stattfinden müssen, aber gerade nichts zur verfügung steht .... folge -> Image-Verlust und Ansehen bei der Betty wird niedriger...

>Denn wir müssen uns ja im Klaren sein, dass die meisten Spieler nach 14 Tagen bei fast 100% Image herum lümmeln und da nur schwer herunter rutschen würden....

>Erfolg != Aufgaben

>Alternativ kann auch diese ganze Sammy-Sache da mit einfließen, wobei ein Letzter Platz bei den Sammys dann ebenfalls einen Verlust bedeuten muss.
>Auch Geldabzug kann eine Folge sein.

>Erfolge sind ja eher derzeitig wie "Archivements", welche nur Positive folgen haben.

>Aufgaben könnten sein:
>Mann muss Filme oder Serien verkaufen (hier muss unterbunden werden, dass keine neuen Filme/Serien gekauft werden, um diese dann zu verkaufen -> Es müssen bereits vorhandene Filme/Serien sein)
>Einschaltquoten erreichen -> 50% dann gibts +1% Image bei nicht erreichen -10% (Beispiel halt)
>Nachrichten ebenfalls eine gewisse Aktualität oder bestimmte Themen intensiver ins Programm aufnehmen.

>Chef bestimmt, dass 1-2 Tage Keine Krimis/Komödien etc gesendet werden dürfen, sonst gibt es halt -10% Image (OHNE Positive Belohnung, ist schließlich Anweisung vom Chef)

>Chef bestimmt, dass 7 Tage nichts neues eingekauft werden darf (sollte natürlich erst nach 7 Spieltagen frühestens ausgesprochen werden können oder halt wenn 2 Tage in Folge +/- 0€ Kontostand wäre)

>usw.
>also quasi wirklich bissl um den Schwierigkeitsgrad zu erhöhen bzw. den Chef auch mal etwas sagen zu lassen.
>ODER: 1 Tag Werbeverbot -> + Image oder - Image & Geldstrafe
>also da kann ich mir ne Menge vorstellen, wie sowas dann anders als Archivements gewertet werden kann
>hab bei GitHub mal gelesen :wink: bin dort ja nicht angemeldet ^^

**Comment by nittka** [comment]
Ja, das passt zum Thema Langzeitspielspaß (siehe auch https://github.com/TVTower/TVTower/issues/126#issuecomment-723619396). Um nochmal kurz zum Thema "Abgrenzung Aufgabe vs Erfolg" zu kommen. Technisch ist der Unterschied schon klar, man muss nur aufpassen, dass eine Aufgabe nicht gleichzeitig ein Erfolg ist. Zumindest erscheint es mir nicht sinnvoll, wenn man für eine Chef-Aufgabe 5Mio Zuschauer gleichzeitig auch noch ein Prämie für den entsprechenden Erfolg kassiert.

D.h. existierende Erfolge, ggf. neue Erfolge (#174) und mögliche Aufgaben (siehe oben und mehr) sammeln und in den richtigen Topf tun.

**Comment by StukovTTV** [comment]
Eine stärkere Einbindung des Chefs finde ich auch sehr gut, derzeit ist der in den meisten Spielen ja ziehmlich überflüssig :)

**Comment by GWRon** [comment]
Verbindung zu #558

**Comment by nittka** [comment]
Chef-spezifische Spielziele, Erfolge und Missionen sind zwar verwandt aber nicht dasselbe.
Eine Mission wird vor Spielstart gewählt, ändert sich nicht und das Spiel endet, wenn die Mission erfüllt ist (oder nicht mehr erfüllt werden kann).

Es spricht auch nichts dagegen, alle Varianten zu haben (wobei die höchste Prio für mich die Missionen wären, damit es nicht mehr nur ein Endlosspiel ohne Ziel gibt). Auf diesem Bildschirm könnte angezeigt werden, was die aktuelle Mission ist. Wenn man mehrere Spiele gestartet hat, kann man nach dem Laden eines Spielstands nochmal nachlesen, was zu tun ist. Eine Toastnachricht nach dem Laden fände ich trotzdem angebracht.

Wenn es Chef-Spielziele/Aufträge gibt, könnten auch die auf dem Bildschirm auch nochmal erscheinen. Allerdings finde ich, dass die Auftragserteilung beim Chef selbt erfolgen sollte.

Die Frage ist, ob das den Bildschirm wirklich ausfüllt. Die Missionsbeschreibung und Chefaufträge passen im besten Fall jeweils in eine Zeile. Man könnte hier natürlich auch noch die gewonnenen Sammys auflisten (durch die riesige Sprechblase beim Chef ist dort wenig Platz, um was an die Wand zu heften).
